woocommerce-gateway-ebanx/gateways/class-wc-ebanx-credit-card-gateway.php

```diff
@@ -402,20 +407,22 @@ public function get_payment_terms($amount, $max_instalments, $tax = 0) {
 
 			if (isset($instalment_taxes) && array_key_exists($number, $instalment_taxes)) {
 				$cart_total += $cart_total * $instalment_taxes[$number];
+				$cart_total += $cart_total * $tax;
 				if ($instalment_taxes[$number] > 0) {
 					$has_interest = true;
 				}
 			}
 
-			$instalment_price = $cart_total / $number;
-			$instalment_price += $instalment_price * $tax;
-			$instalment_price = round(floatval($instalment_price), 2);
+			if ( $this->is_valid_instalment_amount($cart_total, $number) ) {
+				$instalment_price = $cart_total / $number;
+				$instalment_price = round(floatval($instalment_price), 2);
```

Comment:
`floatval` of division result, when stored in a variable if it would break php and cast to integer it would be already too late... so no need to `floatval` here... it will work nonetheless.